applications/RANSModellingApplication/RANSModellingApplication.py

```diff
@@ -0,0 +1,15 @@
+# makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+from __future__ import print_function, absolute_import, division
+
+# Application dependent names and paths
+from KratosMultiphysics.kratos_utilities import CheckIfApplicationsAvailable
+if (CheckIfApplicationsAvailable("TrilinosApplication")):
```

Comment:
I know this is necessary, but I would at least also check for an MPI-run
```suggestion
if KratosMultiphysics.IsDistributedRun() and CheckIfApplicationsAvailable("TrilinosApplication"):
```

Reply:
I cannot do this, It complains some of the symbols not found, eventhough I'm running a non-mpi test :/